cmd/registry-disk-v1alpha/Dockerfile

```diff
@@ -16,13 +16,13 @@
 # Copyright 2017 Red Hat, Inc.
 #
 
-FROM debian:sid
+FROM fedora:28
 
 LABEL maintainer="David Vossel <dvossel@redhat.com>"
 ENV container docker
 
-RUN apt-get update && apt-get install -y \
-	bash curl bzip2 qemu-utils && \
+RUN dnf install -y \
+	bash curl bzip2 qemu-img && \
 	mkdir -p /disk && \
 	rm -rf /var/lib/apt/lists/*
```

Comment:
with dnf this line is obsolete, and cleanup needs to be done different. Also, this change is already in https://github.com/kubevirt/kubevirt/pull/1719

Reply:
Once yum is cleand up with `&& dnf -y clean all`, like suggested by @slintes, lgtm.